language-support/qute/qute.tmLanguage.json

```diff
@@ -245,6 +248,35 @@
         }
       ]
     },
+    "section_start_case_is_tag": {
+      "begin": "({)(#)((case|is)\\b)",
+      "end": "(})",
+      "beginCaptures": {
+        "1": {
+          "name": "support.constant.qute"
+        },
+        "2": {
+          "name": "keyword.control"
+        },
+        "3": {
+          "name": "keyword.control"
+        }
+      },
+      "endCaptures": {
+        "1": {
+          "name": "support.constant.qute"
+        }
+      },
+      "patterns": [
+        {
+          "match": "(?<={(#case|#is)\\s)(gt|ge|lt|le|not|ne|in|ni|!in|>|>=|<|<=|!=)(?=\\s\\w+)",
```

Comment:
If I do:

```
{#case  gt 10}
```
or
```
{#case gt     10}
```
then the highlighting doesn't work properly. I think it shouldn't be too hard to handle these cases.